wp-content/themes/core/components/follow/Follow_Controller.php

```diff
@@ -51,22 +50,22 @@ public function get_social_links(): Social_Link_Collection {
 
 	protected function get_label( string $site ): string {
 		switch ( $site ) {
-			case Social_Settings::FACEBOOK:
+			case Theme_Options::SOCIAL_FACEBOOK:
 				return __( 'Like us on Facebook', 'tribe' );
```

Comment:
```suggestion
				return esc_html__( 'Like us on Facebook', 'tribe' );
```

Might as well fix all of these while we're in here, the ones below also need the same treatment.